app/views/houses/show.html.erb

```diff
@@ -1,7 +1,7 @@
-<h1><%= @house.name %></h1>
-<% @house.students.each do |student| %>
+<h1><%= @houses.name %></h1>
+<% @house.Students.each do |student| %>
```

Comment:
You can say `@house.students` (lowercase s). If you capitalize it, it looks for a class called `Students`, which does not exist.